sql/src/main/java/io/crate/operation/reference/information/InformationTablePartitionsExpression.java

```diff
@@ -30,6 +30,8 @@
 abstract class InformationTablePartitionsExpression<T>
     extends RowContextCollectorExpression<PartitionInfo, T> {
 
+    public static final BytesRef tableType = new BytesRef("BASE TABLE");
```

Comment:
Think it's okay to keep it hard-coded here for now. Wouldn't anticipate too much here. We might end up *not* using TableInfo if we add views for example. So I'd keep it simple for now as it's easy to change this way, but much harder if we add it to TableInfo and it ends up being wrong there.